content/en/docs/setup/production-environment/tools/kubeadm/enable-disable-dual-stack.md

```diff
@@ -0,0 +1,130 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+- neolit123
+- sftim
+title: Enable Or Disable Dual-Stack via kubeadm
```

Comment:
ok, give this is a feature gate it means that the feature at some point will be locked to true and cannot be disabled.
with that in mind, IMO we should name the page more broadly:

```suggestion
title: Dual-stack support with kubeadm
```

```
dual-stack-support.md
```

once the feature is GA, we can keep some of the details, but remove the feature gate related toggles.